src/app/data-explorer/data-request-row/data-request-row.component.ts

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
Personally, I liked the view details button, but this works well. I will need to make a task to update the new layout for the query builder to match the icons you have used over grey buttons.